playbooks/roles/thoth-core/tasks/main.yaml

```diff
@@ -40,12 +40,6 @@
   - name: create Thoth Core configmap template
     command: oc create --namespace "{{ THOTH_INFRA_NAMESPACE  }}" --filename ../openshift/configmap-template.yaml
  
-  - name: create Thoth Package Releases CronJob template
```

Comment:
where do we create the templates now?

Reply:
Ah, forgot to add changes - should be fixed now.

Reply:
Wait... :)